lib/Basic/PlatformUtility.cpp

```diff
@@ -0,0 +1,27 @@
+//===- Support/PlatformUtility.cpp - Platform Specific Utilities ----------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#include "llbuild/Basic/PlatformUtility.h"
+
+#if defined(_WIN32)
```

Comment:
Can we define WIN32_LEAN_AND_MEAN here, and maybe have a local Windows compatibility header which does that?